.bazelrc

```diff
@@ -194,7 +202,7 @@ build:linux --force_pic
 
 # On Linux, garbage collection sections and optimize binary size. These do not apply to the macOS
 # toolchain.
-build:linux --linkopt="-Wl,--gc-sections" --linkopt="-Wl,-O2"
+build:linux --linkopt="-Wl,--gc-sections"
```

Comment:
Is this change mentioned in the PR description?

Reply:
Good catch. It was mentioned in the description of the second commit, which I added after the original PR. Updated the PR description too.